docs/user-guides/sbs_swarm_interface.md

```diff
@@ -0,0 +1,477 @@
+
+# Prerequisites
+
+We will assume that you already know this before you start with the tutorial:
+
+* Some basic experience with python
+* Followed the [crazyflie getting started guide](https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/) 
+* Read the [high level commander](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/crazyflie/high_level_commander/), [swarm](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/crazyflie/swarm/) and [SyncCrazyflie](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/crazyflie/syncCrazyflie/) documentation .
+
+
+# Get the script started
+
+Since you should have installed cflib in the previous step by step tutorial, you are all ready to got now. Open up an new python script called `swarm_rectangle.py`. First you will start by adding the following import to the script:
+
+```python
+import time
+
+import cflib.crtp
+from cflib.crazyflie.swarm import CachedCfFactory
+from cflib.crazyflie.swarm import Swarm
+
+uris = {
+    'radio://0/20/2M/E7E7E7E701',
+    'radio://0/20/2M/E7E7E7E702',
+    'radio://0/20/2M/E7E7E7E703',
+    'radio://0/20/2M/E7E7E7E704',
+    # Add more URIs if you want more copters in the swarm
+}
+
+if __name__ == '__main__':
+    cflib.crtp.init_drivers()
+    factory = CachedCfFactory(rw_cache='./cache')
+    with Swarm(uris, factory=factory) as swarm:
+```
+
+This will import all the necessary modules and open the necessary links for communication with all the Crazyflies of the swarms.`Swarm` is a wrapper class which facilitates the execution of functions given by the user for each copter and can execute them in parallel or sequentially.Each Crazyflie is treated as a `SyncCrazyflie` instance and as the first argument in swarm wide actions.There is no need to worry about threads since they are handled internally.To reduce connection time,the factory is chosen to be instance of the `CachedCfFactory` class that will cache the Crazyflie objects in the `./cache` directory .
+
+The radio addresses of the copters are defined in the `uris` list and you can add more if you want to.
+
+# Step 1: Light Check
+
+In order to verify everything is setup and working properly a light check will be performed.During this check,all the copters will light up red for a short period of time and then return to normal.
+This is achieved by setting the parameter `led.bitmask` to 255 which results to all the LED's of each copter light up simultaneously.
+
+Add the helper functions `activateBitMask`,`deactivateBitMask` and the function`lightCheck` above `__main__`:
+```python
+def activateBitMask(scf):
```

Comment:
These functions are not snake-case which is different from the names in the next section, should be the same I guess?
Also possibly rename to `activate_led_bit_mask` to make it clearer that it is related to LEDs and not any general bitmask.